api/v1/index.js

```diff
@@ -2,12 +2,12 @@ const { Router } = require('express');
 
 const router = Router();
 
-// const accounts = require('./accounts');
+const accounts = require('./accounts');
 const chars = require('./chars');
 const items = require('./items');
 const misc = require('./misc');
 
-// router.use('/accounts', accounts);
```

Comment:
This still needs to be commented out, since the account management section should remain disabled for now.

Reply:
Accounts will be back in with this PR, login and account creation are handled on the same page of /accounts